ppx_mysql_async.opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Dario Teixeira <dte@issuu.com>"
+author: "Team Raccoon"
+homepage: "https://github.com/issuu/ppx_mysql"
+dev-repo: "https://github.com/issuu/ppx_mysql.git"
+bug-reports: "https://github.com/issuu/ppx_mysql/issues"
+doc: "https://issuu.github.io/ppx_mysql/"
+available: [ ocaml-version >= "4.03.0" ]
+build: [["dune" "build" "-p" name "-j" jobs]]
+build-test: [["dune" "runtest" "-p" name]]
+depends: [
+  "async" {>= "v0.11" & < "v0.12"}
+  "dune" {build}
+  "mysql" {>= "1.2" & < "2.0"}
+  "ppx_mysql" {= "0.1"}
```

Comment:
I'm a little bit worried about this, because this will be part of the next release but these constraints need to be consistent otherwise it will be completely unusable. I wonder if there is a way to specify "all of them need to be the same version"